chain/consensus/hierarchical/actors/subnet/subnet_state.go

```diff
@@ -80,6 +79,10 @@ type SubnetState struct {
 	Checkpoints cid.Cid // HAMT[epoch]Checkpoint
 	// WindowChecks
 	WindowChecks cid.Cid // HAMT[cid]CheckVotes
+	// Validators contains information about BFT validators
+	Validators []Validator
+	// ValidatorsNumber initial number of validators to start a subnet with BFT-type consensus
+	ValidatorsNumber uint64
```

Comment:
MinValidators? (the current number of validators can be checked through `len(Validators)` (see comment above)